<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1141</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="label_src_path">
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_dst_path">
        <property name="styleSheet">
         <string notr="true">border: 1px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_src_path">
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dst_path">
        <property name="text">
         <string>目标文件夹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>660</y>
     <width>226</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_quit">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_startc">
      <property name="text">
       <string>转换</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>711</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black</string>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件路径</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black</string>
   </property>
   <property name="text">
    <string>图片预览:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lable_image_pre">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>180</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>590</y>
     <width>85</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>提示框:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>590</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
